CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **AWSIoT**
+  - IoT update for retain feature ([PR #3681](https://github.com/aws-amplify/aws-sdk-ios/pull/3681))
```

Comment:
Suggest using the "retained messages" term of art to help discoverability. Also check with @hdworld11 -- we might be able to lift something from the launch announcement, if any exists.

```suggestion
  - AWSIoT now supports retained messages ([PR #3681](https://github.com/aws-amplify/aws-sdk-ios/pull/3681))
```

Reply:
updated

Reply:
I'm not seeing this is updated?